src/papi.c

```diff
@@ -879,27 +884,30 @@ PAPI_list_threads( PAPI_thread_id_t *tids, int *number )
  *
  *	@par Example:
  *	@code
- int ret;
- RateInfo *state = NULL;
- ret = PAPI_thread_init(pthread_self);
- if (ret != PAPI_OK) handle_error(ret);
- 
- // Do we have the thread specific data setup yet?
-
-ret = PAPI_get_thr_specific(PAPI_USR1_TLS, (void *) &state);
-if (ret != PAPI_OK || state == NULL) {
-	state = (RateInfo *) malloc(sizeof(RateInfo));
-	if (state == NULL) return (PAPI_ESYS);
-	memset(state, 0, sizeof(RateInfo));
-	state->EventSet = PAPI_NULL;
-	ret = PAPI_create_eventset(&state->EventSet);
-	if (ret != PAPI_OK) return (PAPI_ESYS);
-	ret = PAPI_set_thr_specific(PAPI_USR1_TLS, state);
-	if (ret != PAPI_OK) return (ret);
-}
-*	@endcode
-*	@see PAPI_register_thread PAPI_thread_init PAPI_thread_id PAPI_set_thr_specific
-*/
+ * int ret;
+ * RateInfo *state = NULL;
+ * ret = PAPI_thread_init(pthread_self);
+ * if (ret != PAPI_OK) handle_error(ret);
+ * 
+ * // Do we have the thread specific data setup yet?
+ * 
+ * ret = PAPI_get_thr_specific(PAPI_USR1_TLS, (void *) &state);
+ * if (ret != PAPI_OK || state == NULL) {
+ *     state = (RateInfo *) malloc(sizeof(RateInfo));
+ *     if (state == NULL) return (PAPI_ESYS);
+ *     memset(state, 0, sizeof(RateInfo));
+ *     state->EventSet = PAPI_NULL;
+ *     ret = PAPI_create_eventset(&state->EventSet);
+ *     if (ret != PAPI_OK) return (PAPI_ESYS);
+ *     ret = PAPI_set_thr_specific(PAPI_USR1_TLS, state);
+ *     if (ret != PAPI_OK) return (ret);
+ * }
+ *	@endcode
+ *	@see PAPI_register_thread
```

Comment:
unneeded indentation?

Reply:
All the tags must be at the same level of indentation for it to render correctly. Then I will have to dedent all the tags down.